tap_salesforce/client.py

```diff
@@ -13,11 +13,12 @@
     TapSalesforceOauthException,
     TapSalesforceQuotaExceededException,
     TapSalesforceInvalidCredentialsException,
+    QueryLengthExceedLimit,
     build_salesforce_exception,
 )
 from tap_salesforce.metrics import Metrics
 
-MAX_CUSTOM_FIELDS = 450
+MAX_QUERY_LENGTH = 10000
```

Comment:
Did you verify that they handle 10k length? I thought it was like, 8k.

Reply:
I tested it. It seems fine